Automattic/jetpack
# Untangle: Update Social Connections Link

Fixes https://github.com/Automattic/dotcom-forge/issues/5552

## Proposed changes:

* This PR updates the Jetpack Social link & button to redirect to the Jetpack Cloud Social Connections page for sites with access to Jetpack Cloud. (Note that proxied Atomic sites with the Classic admin interface enabled are now granted access to Jetpack Cloud via D138559-code)
* This PR also updates the link copy from "WordPress.com sharing page" to "Connections page". This way the copy makes sense whether it redirects to Calypso or Jetpack Cloud.

Atomic Site WP-Admin
Before | After
--|--
<img width="1526" alt="classic-before" src="https://github.com/Automattic/jetpack/assets/140841/d17798a6-be42-4c76-9093-1c06c8c465e0"> | <img width="1523" alt="classic-after" src="https://github.com/Automattic/jetpack/assets/140841/3edd3b40-147c-4026-ae83-c1ffd86cc38c">

Jetpack Site
Before | After
--|--
<img width="1526" alt="jetpack-before" src="https://github.com/Automattic/jetpack/assets/140841/704095c1-dd62-4969-af81-172be8e9bb2f">  | <img width="1526" alt="jetpack-after" src="https://github.com/Automattic/jetpack/assets/140841/285fae8f-e76b-4440-9e1c-40d3ef5f4101">





### Other information:


## Jetpack product discussion


p1708126615252529-slack-CDLH4C1UZ

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Use the Jetpack Beta plugin to load this PR.
* On a Jetpack site goto /wp-admin/options-general.php?page=sharing click on the "Connections page" link and "Jetpack Social Settings" button highlighted in the picture above. The links should redirect you to the Jetpack Cloud "Social Connections" page at https://cloud.jetpack.com/jetpack-social/[site-id]
* The above should also be true on an Atomic site with Classic admin enabled while you're proxied.
* Test the links above un-proxied. They should now redirect you to Calypso https://wordpress.com/marketing/connections/[site-id]?site=[site-id]
* Now test proxied, but with "Default" admin view enabled. It should redirect you to Calypso https://wordpress.com/marketing/connections/[site-id]?site=[site-id]
